test/test_helper.rb

```diff
@@ -5,3 +5,9 @@
 require "common/factory_tests"
 require "psych"
 require "pry-byebug"
+
+def assert_close_enough(a, b)
+  delta = Math.sqrt(a * a + b * b) * 0.00000001
+  delta = 1e-7 if delta < 1e-7
```

Comment:
This seems smart but IDK why, could you explain it to me ? :)

And maybe add a method comment I guess

Reply:
Method was already there, I've just moved it :).
My understanding is that it's computing some kind of distance to make it positive. But I have to admit it kind of looks odd.

Reply:
I've documented the method and simplified it